src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -38,4 +40,12 @@ public Driver update(Driver driver) {
     public boolean delete(Long id) {
         return driverDao.delete(id);
     }
+
+    @Override
+    public Optional<Driver> findByLogin(String login) throws AuthenticationException {
```

Comment:
Remove the 'throws AuthenticationException' from the method signature since it is not thrown in the method implementation.